Dockerfile

```diff
@@ -48,6 +48,9 @@ COPY web/package-lock.json web/package.json \
 ARG NEXT_PUBLIC_AGREEMENT_TIMER=180 # Seconds required before agreements can be agreed to
 ENV NEXT_PUBLIC_AGREEMENT_TIMER=$NEXT_PUBLIC_AGREEMENT_TIMER
 
+ARG NEXT_PUBLIC_ENABLE_PROJECTS=true # Enable projects feature
+ENV NEXT_PUBLIC_ENABLE_PROJECTS=$NEXT_PUBLIC_ENABLE_PROJECTS
+
```

Comment:
I added this to get the projects tests to run during CI (otherwise the projects feature is not available). But I'm not sure how you feel about this @t-young31? Is there an alternative way to approach this?